scripts/windows/apktool.bat

```diff
@@ -3,6 +3,12 @@ setlocal
 set BASENAME=apktool_
 chcp 65001 2>nul >nul
 
+set java_exe=java.exe

```

Comment:
On my windows computer, I never need to enter `.exe`. I just type in `java` at command line and it works. It appears both `java` and `java.exe` works